xbmc/windowing/gbm/WinSystemGbmGLContext.h

```diff
@@ -35,7 +35,7 @@ class CWinSystemGbmGLContext : public CWinSystemGbmEGLContext, public CRenderSys
   bool SetFullScreen(bool fullScreen, RESOLUTION_INFO& res, bool blankOtherDisplays) override;
   void PresentRender(bool rendered, bool videoLayer) override;
 protected:
-  void SetVSyncImpl(bool enable) override { return; };
+  void SetVSyncImpl(bool enable) override {};
```

Comment:
kill the ;'s while at it ?

Reply:
done, now clang-format formatted those lines as expected